docs/UserGuide.md

```diff
@@ -398,10 +409,10 @@ Expected output:
 ### <a id="add-e"><ins>3.16 Add Event:</ins> `add-e`</a>
 Adds an event to the list.
 
-Format: `add-e /t TITLE /timing DATETIME [/repeat REPEAT] [/stop REPEAT_END] [/remind REMIND]`
+Format: `add-e /t TITLE /timing DATETIME [/repeat REPEAT] [/stop REPEAT_END] [/remind REMIND] [/tag TAG_1] [/tag TAG_2]...`
 
-💡 DATETIME format pattern "dd-MM-yyyy HH:mm:ss”
-💡 Specifying  [/repeat RECURRING] will set the event as a recurring event.
+💡 DATETIME format pattern "dd-MM-yyyy HH:mm:ss” <br>
+💡 Specifying [/repeat RECURRING] will set the event as a recurring event. <br>
```

Comment:
Not sure if the line break is necessary, don't think some of it has? Should probably standardize